packages/hooks/src/useCurrencyExchangeRate.ts

```diff
@@ -0,0 +1,10 @@
+import { useStore } from '@deriv/stores';
+
+const useCurrencyExchangeRate = (currency: string) => {
```

Comment:
Can you help to write a documentation above this hook and explain about its functionality.
Perhaps provide an example as well.

Reply:
done :heavy_check_mark: